GliaWidgets/View/Call/CallView.swift

```diff
@@ -32,6 +32,7 @@ class CallView: EngagementView {
     private let kRemoteVideoViewPortraitHeightMultiplier: CGFloat = 0.3
     private let kRemoteVideoViewLandscapeHeightMultiplier: CGFloat = 1.0
     private let kBarsHideDelay: TimeInterval = 3.2
+    private let barsAreHidden = ObservableValue<Bool>(with: false)
```

Comment:
should we dispose it on deinit to prevent leak?

Reply:
Wait, why do we need ObservableValue in the first place

Reply:
hm, to be on the safe side - probably yes. will fix shortly
Unfortunately, there are _a lot_ other `ObservableValue`s in the project that are not being disposed.

Reply:
> Wait, why do we need ObservableValue in the first place

now that you are telling this, I understand that this could've been just a Bool. Whoops